### ISSUE TITLE
Next steps

### ISSUE THREAD

**Issue by pmelchor** [citation-style-language/citation-style-language.github.io#1]
@rmzelle Here is the plan: please let me know if it sounds good to you.

I am going to be posting the following as issues:

- A project status (with "**info**" label)
- A few to-dos that I must complete before we can migrate CitationStyles.org to GitHub Pages ("**must**" label)
- And a few other to-dos that are improvements which can be done after the migration ("**nice-to-have**" label)

Can you give me the necessary permissions so that I can close issues as I finish my work on them?

Once I have posted all of the above I will ask you to review the website and add any other issues following the same "must" vs "nice-to-have" convention.

**Comment by rmzelle** [comment]
> Can you give me the necessary permissions so that I can close issues as I finish my work on them?

I think that GitHub users can always close their own issues.

**Comment by pmelchor** [comment]
Yes, I was actually thinking of the second part we will eventually get to:

> Once I have posted all of the above I will ask you to review the website and add any other issues following the same "must" vs "nice-to-have" convention.

We are good for now :-)

**Comment by rmzelle** [comment]
@pmelchor, how's life? I was wondering if you will have some time in the next few months to help us complete the migration from WordPress to GitHub Pages. (if not, that's fine too)

**Comment by rmzelle** [comment]
You already put quite some effort in getting the Minimal Mistakes Jekyll theme ready, but just in case we get stuck, I've also been quite happy with mkdocs for my day job, which has some nice themes as well: http://squidfunk.github.io/mkdocs-material/.

**Comment by pmelchor** [comment]
Hi @rmzelle , sorry for my prolonged absence. Whenever I see CSL news I wish I could be contributing regularly, but with my business + academic + family obligations I have just not been able to fit in the extra project (even though it is something I have enjoyed a lot when I have been able to put in the time).

The best I could offer right now is to make the migration my Christmas break project (with academic work out of the way for 10 days or so). If you have someone who can do this earlier I would be happy to provide some guidance and support.

**Comment by bdarcus** [comment]
Might be worth looking into Hugo?

https://gohugo.io/

https://gohugo.io/hosting-and-deployment/hosting-on-github/

**Comment by rmzelle** [comment]
> The best I could offer right now is to make the migration my Christmas break project (with academic work out of the way for 10 days or so). If you have someone who can do this earlier I would be happy to provide some guidance and support.

That would be great. We don't have a lot of HTML/CSS experts on staff, but I can look at some of the tickets you filed and see if I can address them before then.

> Might be worth looking into Hugo?

@pmelchor already put in quite a bit of effort in migrating the site to the Jekyll theme, so we should probably first see if can make things work with that framework. Jekyll also has the advantage that GitHub Pages autobuilds it.

**Comment by adam3smith** [comment]
Also, while I'm nowhere close to Pablo's level, I'm pretty comfortable with Jekyll and would be able to maintain & fix small issues -- so lots of reasons to stick with it.

And I agree, Christmas break sounds great for us.

**Comment by pmelchor** [comment]
OK, time blocked for Christmas break :-)

@bdarcus I have looked a little bit into Hugo, thanks! It looks great but, besides what Rintze and Sebastian have said, I see that the Hugo community right now is still quite small compared to Jekyll's —which is already pretty small :-) —. Since we are delving into the rather niche world of static website generators, I think we may want to stick to the most popular so that it is easier to get community support or eventually have other contributors give us a hand.

@rmzelle I have made a note to compare the outputs of mkdocs and Minimal Mistakes.
